apps/chat/src/store/conversations/conversations.selectors.ts

```diff
@@ -824,3 +824,8 @@ export const selectLastConversationSettings = createSelector(
   [rootSelector],
   (state) => state.lastConversationSettings,
 );
+
+export const selectRenamingConversation = createSelector(
+  [rootSelector],
+  (state) => state.renamingConversation,
+);
```

Comment:
I don't think only Id in state enough and selector which find conversation by this id
```suggestion
const selectRenamingConversationId = createSelector(
  [rootSelector],
  (state) => state.renamingConversationId
);

export const selectRenamingConversation = createSelector(
  [selectConversations, selectRenamingConversationId],
  (conversations, renamingConversationId) => conversations.find(conv => conv.id === renamingConversationId),
);
```